OpenOversight/app/auth/views.py

```diff
@@ -281,10 +281,7 @@ def change_dept():
 @auth.route("/users/", methods=[HTTPMethod.GET])
 @admin_required
 def get_users():
-    if request.args.get("page"):
-        page = int(request.args.get("page"))
-    else:
-        page = 1
+    page = int(request.args.get("page", 1))
```

Comment:
This one line takes the place of those three. The second parameter is the default value should `"page"` not exist in the args.